ui/app/rif/components/confirmationMessage/confirmationMessage.js

```diff
@@ -0,0 +1,57 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+import Modal from 'react-modal'
+import PropTypes from 'prop-types'
+import rifActions from '../../actions'
+
+class ConfirmationMessage extends Component {
+
+  static propTypes = {
+    message: PropTypes.object,
+    opened: PropTypes.bool,
+    dispatch: PropTypes.func,
+  }
+
+  afterOpenModal () {
```

Comment:
If it's not used, it need to be here? It's required?

Reply:
I kept this function as an informative function, something that later if we want to use it we know it's there, because React Modal doesn't have a good doc on that but the function exists and it's there, we can remove it if you want but i just kept that there because i thought it could be useful in the future.

Reply:
Ok! We can leave this function there, it's good to me.